handlers/main.yml

```diff
@@ -9,6 +9,7 @@
 - name: Restart tinc VPN connection
   shell: test -d /run/systemd/system &&
          ( systemctl restart ifup@{{ tinc_interface }}.service ) ||
+         ( systemctl restart tinc@{{ tinc_network }}.service) ||
          ( ifdown {{ tinc_interface }} ; ifup {{ tinc_interface }} )
```

Comment:
I believe that with this change `debops.tinc` will break in non-systemd based systems. If `systemd` is not detected at runtime, Ansible will execute `systemctl restart tinc@{{ tinc_network }}.service` command which won't be valid. And I'm not sure what state the code will be in after that. Also, is a separate `tinc` unit needed, when `tinc` is controlled by `ifupdown`? Needs more testing.